README.md

```diff
@@ -52,13 +64,56 @@ $ conda install cylc-uiserver
 $ pip install cylc-uiserver
 ```
 
-## Running
+### For Multi-User Setups
 
 ```console
-$ cylc hub
+# via conda (preferred)
+$ conda install cylc-uiserver-hub
+
+# via pip (consult jupyterhub documentation)
+$ pip install cylc-uiserver[hub]
+```
+
+
+## Running
+
+The Cylc UIServer is a
+[Jupyter Server](https://github.com/jupyter-server/jupyter_server)
+extension (like [JupyterLab](https://github.com/jupyterlab/jupyterlab)).
```

Comment:
Ah, I guess it's an extension then, I thought it was an application :+1:

Reply:
If splitting hairs it is technically an "extension application"!

From the users perspective the difference between an "extension" and an "extension application" is invisible.

Reply:
Brilliant, thanks @oliver-sanders ! It was more so I knew the correct term in case I needed to report an issue upstream, or talk to someone else about Jupyter Server. But extension application is quite a mouthful, I'll probably just use jupyter server, or jupyter server app, or whichever my :brain: picks first.